x-pack/plugins/observability/public/hooks/use_fetch_rules.ts

```diff
@@ -39,6 +45,7 @@ export function useFetchRules({ searchText, ruleLastResponseFilter, page, sort }
         searchText,
         typesFilter: OBSERVABILITY_RULE_TYPES,
         ruleStatusesFilter: ruleLastResponseFilter,
+        ruleStatusFilter: [false],
```

Comment:
I hardcoded here just for testing. ruleStatusFilter at the moment is an array of strings for example ['enabled', 'disabled'], but `alert.attributes.enabled` is a boolean, so these values need to be converted to booleans.

With the newly added snoozed status, the logic needs to be changed, that's why I didn't proceed further and hardcoded some boolean value to see it working. This implementation will definitely change.